libevmasm/CommonSubexpressionEliminator.cpp

```diff
@@ -91,32 +91,32 @@ void CommonSubexpressionEliminator::optimizeBreakingItem()
 	{
 		AssemblyItem::JumpType jumpType = m_breakingItem->getJumpType();
 
-		Id condition = m_state.stackElement(m_state.stackHeight() - 1, itemLocation);
+		Id condition = m_state.stackElement(m_state.stackHeight() - 1, yul::DebugData::create(itemLocation));
```

Comment:
A few lines up instead of fetching ``itemLocation``, we shoudl really just fetch ``itemDebugData = m_breakingItem->debugData();`` and here just pass along that same shared pointer, shouldn't we? Like you already do below in the other cases.